conf/application.conf

```diff
@@ -1,3 +1,27 @@
 # https://www.playframework.com/documentation/latest/Configuration
 play.http.secret.key = "QCY?tTynq?aZ?iwrNw?xIlR6CTf:G3gf:90Latabg@5241AB`R5W:1uDFN];Ik@n"
 play.http.secret.key = ${?APPLICATION_SECRET}
+
+# Default database config, PostgreSQL
+db.default.driver=org.postgresql.Driver
+db.default.url="postgres://postgres:pass@localhost:5432/pass_bakery_db"
+
+# Number of database connections
+# See https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing
+fixedConnectionPool = 9
+
+play.db {
+  prototype {
+    hikaricp.minimumIdle = ${fixedConnectionPool}
+    hikaricp.maximumPoolSize = ${fixedConnectionPool}
+  }
+}
+
+# Job queue sized to HikariCP connection pool
+database.dispatcher {
+  executor = "thread-pool-executor"
+  throughput = 1
+  thread-pool-executor {
+    fixed-pool-size = ${fixedConnectionPool}
+  }
+}
```

Comment:
Currently you are specifyng the default configurations for evolutions. it is best practice to specify everything you need. Not a blocker but do read about what are the default values

Reply:
No changes yet. This code is from when I was setting up the execution context, it was brought from [Play Framework Anorm Example](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-anorm-example) 
path: `conf/application.conf`